src/renderers/Renderer.ts

```diff
@@ -70,6 +70,7 @@ export class Renderer {
     const platform = voxaEvent.platform.name;
 
     let message = voxaEvent.t(view, {
+      defaultValue: view,
```

Comment:
When sending a `string` to the key for the directives, Voxa thinks is a path to be rendered by i18n. The value returned is super random since i18n is really trying to render something. When setting the `defaultValue` flag, i18n will return the object that it couldn't translate, and that's exactly what we need for the directives in case the value sent is a string.

@armonge